lemkus_spider.py

```diff
@@ -0,0 +1,59 @@
+import json
+import re
+import scrapy
+
+
+class LemkusSpider(scrapy.Spider):
+    name = "lemkus-spider"
+    start_urls = [
+        'https://lemkus.com/collections/sneakers',
+        'https://lemkus.com/collections/apparel',
+        'https://lemkus.com/collections/kids'
+    ]
+
+    @staticmethod
+    def get_current_page_no_to_scrap(response):
+        current_page_url = response.css('link[rel="canonical"]::attr(href)').extract()[0].split('?')
+        if len(current_page_url) < 2:
+            return 1

```

Comment:
We discussed this thing in weatherman as well, conditions against hard coded numeric values are not a good thing. Refactor your logic.